app/Models/V2/User.php

```diff
@@ -321,4 +321,17 @@ public function wipeData()
         $this->instagram = null;
         $this->avatar = null;
     }
+
+    public function getFullNameAttribute(): string
+    {
+        if (empty($this->first_name) && empty($this->last_name)) {
+            return 'Unnamed User';
+        } elseif (empty($this->first_name)) {
+            return $this->last_name;
+        } elseif (empty($this->last_name)) {
+            return $this->first_name;
+        }
+
+        return $this->first_name . ' ' . $this->last_name;
+    }
```

Comment:
This attribute is fine, but is it used in this PR somewhere?

Reply:
Yes, I am using as `$user->full_name`